src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: React Native agent
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Configuration
+metaDescription: Instructions for installing and configuring the New Relic React Native agent.
+---
+
+With our React Native agent, your team can monitor your hybrid app's performance and identify any errors within your code. Our agent collects crash data, network traffic, and other information for hybrid apps using native components. This allows your team to better understand the health of your app. We empower your team to make more informed development choices by providing insight into the following:
+
+  * Capture JavaScript errors
+  * Network instrumentation
+  * Distributed tracing (currently functional in Android, iOS support coming in the near future)
+  * Tracking console log, warn and error
+  * Promise rejection tracking
+  * Capture interactions and the sequence in which they were created
+  * Pass user information to New Relic to track user sessions
+  * Expo support
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Requirements"
+  >
+Before you install the React Native agent, ensure your app meets these requirements:
+
+  * Android API 21 or higher
+  * iOS 10
+  * React Native version 0.63 or higher
+  * [IOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+  * [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+Native support levels are based on [React Native requirements](https://github.com/facebook/react-native#-requirements).
+  </Collapser>
+</CollapserGroup>
```

Comment:
I'd recommend not using clamshells for these sections. typically clamshells are used mostly for content that is branching, where customers would choose one out of several options, and not look at the rest. If it's content that has to be seen, it's best put into an h2. So for example, I definitely would move 'requirements' into an h2 section instead. For the other sections, it's not immediately obvious to me which of these are mutually exclusive versus a customer having to do several. If that makes sense. If the choices are mutually exclusive (a customer would just pick one of several) that's when I'd put them in a collapser format and basically say 'pick one'. If instead they are tasks where a customer would do one or more of them, I'd make them h2s with an intro to each section making it clear when they'd do that. For example, that might mean adding a sentence like 'If you have x situation, you'll want to do this...' to make it very clear. 

It might also make sense to give some context for each step and how it relates to the other steps. For example, in the install step, it might make sense to do something like "First you'll install the agent, and after that we'll give you some instructions for.... Here's how to install the agent:"